### PR TITLE
JBPM-9412 Not able navigate to Parent from child instance even both a…

### PR DESCRIPTION
**Pull request by bxf12315**
…re active instances

**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[JBPM-9412](https://issues.redhat.com/browse/JBPM-9412)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* paste the link(s) from GitHub here
* link 2
* link 3 etc.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by bxf12315** [comment]
Jenkins run fdb

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ public ProcessInstanceDiagramSummary getProcessInstanceDiagramSummary(ProcessIns
         summary.setSubProcessInstances(getProcessInstancesByParentId(processInstanceKey, processInstanceKey.getDeploymentId()));
 
         if (processInstance.getParentId() != -1) {
-            summary.setParentProcessInstanceSummary(getProcessInstance(new ProcessInstanceKey(processInstanceKey.getServerTemplateId(), "", processInstance.getParentId())));
+            summary.setParentProcessInstanceSummary(getProcessInstance(new ProcessInstanceKey(processInstanceKey.getServerTemplateId(), processInstanceKey.getDeploymentId(), processInstance.getParentId())));
```
Comment (nmirasch) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java:152]: @bxf12315 Could you please add some testing here?

Comment (bxf12315) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java:152]: Done

**Comment by prashuts** [comment]
Jenkins run fdb

**Comment by bxf12315** [comment]
Jenkins run fdb

**Comment by bxf12315** [comment]
Jenkins run fdb

**Review by nmirasch — APPROVED** [review]
LGTM

**Review by prashuts — APPROVED** [review]
LGTM

**Pull request merged**